utils/types/zod.ts

```diff
@@ -138,3 +129,18 @@ export const checkInOutFormSchema = z
       path: ['textFieldAttributes'],
     }
   )
+
+export const newItemFormSchema = itemDefinitionResponseSchema
+  .extend({
+    _id: objectId.optional(),
+    attributes: attributeResponseSchema.optional(),
+    category: categoryResponseSchema.nullable().optional(),
```

Comment:
remove `optional`

Reply:
turns out this needs to be there since I redefined what `category` is in the extend bit